src/quest_manager/views.py

```diff
@@ -372,8 +372,10 @@ def quest_list(request, quest_id=None, template="quest_manager/quests.html"):
     # num_inprogress = QuestSubmission.objects.all_not_completed(request.user).count()
     # num_completed = QuestSubmission.objects.all_completed(request.user).count()
 
+    awaiting_submission = QuestSubmission.objects.filter(is_approved=False, is_completed=True).exists()
```

Comment:
What does "awaiting submission" mean?  Did you mean "awaiting approval"?  Also, this is not filtered for the current user, so it will be getting ALL submissions from every user. (I guess this one really needed a test =D

Reply:
changed name and now added user=request.user to the filter, as well as a test